docs/submission-tutorial.md

```diff
@@ -0,0 +1,4 @@
+# Submission Tutorial
+Todo!
+- Include an example of adding a new mod
+- Include an example of adding a new version
```

Comment:
I don't see the point of having a "todo" sorta file exist and being linked to. It can always be added later on. Clicking on links only to discover "todo" would at least annoy myself.

Reply:
Well, I'm so stubborn about not removing that file (I'm afraid I'd forget to ever add it) that I've just gone and made it.